HHS/Head-Start-TTADP
# Integrate new db schema w/ frontend

## Description of change

There will be less changed files once main is merged into[ TTAHUB-747/initial_db_migration](https://github.com/HHS/Head-Start-TTADP/tree/TTAHUB-747/initial_db_migration).

Fixes to properly edit, save, view, and export an activity report with the new schema. 

## How to test

Create a report and edit it, approve it, send it back. 
Also - view an approved report, export existing report. View other pages and confirm they still work.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-0

## Checklists

### Every PR



### Production Deploy


### After merge/deploy
